tool/.deploy-staging.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+# This script is used by .deploy-staging.yaml
+# Do NOT use this script for any other purposes.
```

Comment:
Please add a note about the gcloud credential setup, how this script gets those variables.